Shared/API/CloudFunction.swift

```diff
@@ -10,65 +10,16 @@ import Foundation
 import Parse
 import Combine
 
-protocol CloudFunction: StatusableRequest {
```

Comment:
The StatusableRequest wasn't doing much for us so I deleted and moved the associated type to the Cloud Function protocol.